.env.example

```diff
@@ -7,6 +7,13 @@ FORKING="true"
 COINMARKETCAP_API_KEY=""
 REPORT_GAS="false"
 ETHERSCAN_API_KEY=""
+
 FORKING="false"
-FORK_CHAINID="46"
-FORK_URL="https://bsc-dataseed.binance.org/"
\ No newline at end of file
+
+# BSC
+FORK_CHAINID="56"
+FORK_URL="https://bsc-dataseed.binance.org/"
+
+# ETH
+FORK_CHAINID="1"
+FORK_URL="<YOUR_FORK_URL>"
```

Comment:
there are duplicates of env keys, consider prefixing the env keys by the network name

Reply:
Yes they are identical, but they are never used at the same time, when the BSC config is used, the ETH config is commented and vice versa. 
We should improve this way for example: 
```suggestion
# BSC fork config
FORK_CHAINID="56"
FORK_URL="https://bsc-dataseed.binance.org/"

# ETH fork config
# FORK_CHAINID="1"
# FORK_URL="<YOUR_FORK_URL>"
```